tcp_check/tests/test_tcp_check.py

```diff
@@ -1,7 +1,9 @@
 # (C) Datadog, Inc. 2010-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+
 from copy import deepcopy
+from unittest import mock
```

Comment:
Probably what makes CI fail:

```suggestion
import mock
```